modules/ossm-supported-configurations.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * ossm-supported-platforms-configurations-3x.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-supported-configurations_{context}"]
+= Supported configurations
+
+The following configurations are supported for the current release of {SMProductName}.
+
+[id="ossm-supported-platforms_{context}"]
+== Supported platforms
+
+The {SMProductName} Operator supports multiple versions of `Istio`. Version {MaistraVersion} {SMProductShortName} control planes are supported on the following platform versions:
+
+ifdef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
+* Red Hat OpenShift Container Platform version 4.14 or later
+endif::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
+ifndef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
+* Red Hat {ocp-product-title} version 4.14 or later
+endif::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
+* {product-dedicated} version 4
+* Azure Red Hat OpenShift (ARO) version 4
+* Red Hat OpenShift Service on AWS (ROSA)
+
+[id="ossm-supported-configurations-networks_{context}"]
+== Supported network configurations
+
+{SMProductName} supports the following network configurations.
+
+* OpenShift-SDN
+* OVN-Kubernetes is available on all supported versions of {ocp-product-title}.
+* Third-Party Container Network Interface (CNI) plugins that have been certified on {ocp-product-title} and passed {SMProductShortName} conformance testing. See link:https://access.redhat.com/articles/5436171[Certified OpenShift CNI Plug-ins] for more information.
+
+[id="ossm-supported-configurations-sm_{context}"]
+== Supported configurations for {SMProductShortName}
+
+ifndef::openshift-rosa,openshift-rosa-hcp[]
+* This release of {SMProductName} is only available on {ocp-product-title} x86_64, {ibm-z-name}, {ibm-power-name}, and Advanced RISC Machine (ARM).
+endif::openshift-rosa,openshift-rosa-hcp[]
+ifdef::openshift-rosa,openshift-rosa-hcp[]
+* This release of {SMProductName} is only available on {ocp-product-title} x86_64.
+endif::openshift-rosa,openshift-rosa-hcp[]
+* Configurations where all {SMProductShortName} components are contained within a single {ocp-product-title} cluster.
+* Configurations that do not integrate external services such as virtual machines.
+* {SMProductName} does not support `EnvoyFilter` configuration except where explicitly documented.
+
+[id="ossm-supported-configurations-kiali_{context}"]
+== Supported configurations for Kiali
+
+* The Kiali console is only supported on the two most recent releases of the Google Chrome, Microsoft Edge, Mozilla Firefox, or Apple Safari browsers.
+* The `openshift` authentication strategy is the only supported authentication configuration when Kiali is deployed with {SMProductName} (OSSM). The `openshift` strategy controls access based on the individual's role-based access control (RBAC) roles of the {ocp-product-title}.
+
+[id="ossm-supported-configurations-webassembly_{context}"]
```

Comment:
I think it's best to remove this section. I'm not yet sure if the 3Scale integration will be supported with service mesh 3, but that can be covered in a feature support table rather than here. WebAssembly modules are a service mesh feature rather than a support platform, so it's a bit out of place here.